# Diff: https://project-skyscraper.com/wp-json/wp/v2/search
# File: api/wp-json/wp/v2/search.json
# Timestamp: 2026-05-29 14:09:43 UTC
# Lines: 302 old -> 302 new
# Beautified: yes
--- old/api/wp-json/wp/v2/search.json
+++ new/api/wp-json/wp/v2/search.json
@@ -1,5 +1,65 @@
 [
   {
+    "id": 1833,
+    "title": "SECURITY LOG",
+    "url": "https://project-skyscraper.com/2026/05/29/sec-log-114327/",
+    "type": "post",
+    "subtype": "post",
+    "_links": {
+      "self": [
+        {
+          "embeddable": true,
+          "href": "https://project-skyscraper.com/wp-json/wp/v2/posts/1833",
+          "targetHints": {
+            "allow": [
+              "GET"
+            ]
+          }
+        }
+      ],
+      "about": [
+        {
+          "href": "https://project-skyscraper.com/wp-json/wp/v2/types/post"
+        }
+      ],
+      "collection": [
+        {
+          "href": "https://project-skyscraper.com/wp-json/wp/v2/search"
+        }
+      ]
+    }
+  },
+  {
+    "id": 1812,
+    "title": "Inbox",
+    "url": "https://project-skyscraper.com/inbox/",
+    "type": "post",
+    "subtype": "page",
+    "_links": {
+      "self": [
+        {
+          "embeddable": true,
+          "href": "https://project-skyscraper.com/wp-json/wp/v2/pages/1812",
+          "targetHints": {
+            "allow": [
+              "GET"
+            ]
+          }
+        }
+      ],
+      "about": [
+        {
+          "href": "https://project-skyscraper.com/wp-json/wp/v2/types/page"
+        }
+      ],
+      "collection": [
+        {
+          "href": "https://project-skyscraper.com/wp-json/wp/v2/search"
+        }
+      ]
+    }
+  },
+  {
     "id": 1802,
     "title": "SECURITY LOG",
     "url": "https://project-skyscraper.com/2026/05/27/sec-log-reboot/",
@@ -220,66 +280,6 @@
         {
           "embeddable": true,
           "href": "https://project-skyscraper.com/wp-json/wp/v2/posts/1680",
-          "targetHints": {
-            "allow": [
-              "GET"
-            ]
-          }
-        }
-      ],
-      "about": [
-        {
-          "href": "https://project-skyscraper.com/wp-json/wp/v2/types/post"
-        }
-      ],
-      "collection": [
-        {
-          "href": "https://project-skyscraper.com/wp-json/wp/v2/search"
-        }
-      ]
-    }
-  },
-  {
-    "id": 1682,
-    "title": "Security Log",
-    "url": "https://project-skyscraper.com/sec-log-211927-full/",
-    "type": "post",
-    "subtype": "page",
-    "_links": {
-      "self": [
-        {
-          "embeddable": true,
-          "href": "https://project-skyscraper.com/wp-json/wp/v2/pages/1682",
-          "targetHints": {
-            "allow": [
-              "GET"
-            ]
-          }
-        }
-      ],
-      "about": [
-        {
-          "href": "https://project-skyscraper.com/wp-json/wp/v2/types/page"
-        }
-      ],
-      "collection": [
-        {
-          "href": "https://project-skyscraper.com/wp-json/wp/v2/search"
-        }
-      ]
-    }
-  },
-  {
-    "id": 275,
-    "title": "SECURITY LOG",
-    "url": "https://project-skyscraper.com/2026/05/18/sec-log-022531/",
-    "type": "post",
-    "subtype": "post",
-    "_links": {
-      "self": [
-        {
-          "embeddable": true,
-          "href": "https://project-skyscraper.com/wp-json/wp/v2/posts/275",
           "targetHints": {
             "allow": [
               "GET"
